docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,250 @@
+longest symbolic token defined by the language specification that appears
+starting at the current input position is lexed, if any.
+
+Not all uses of symbolic tokens within the Carbon grammar will be treated as
+operators. For example, `(` and `)` tokens serves to delimit various grammar
+productions, and we may not want to consider `.` to be an operator, because its
+right "operand" is not an expression.
+
+The presence or absence of whitespace around the symbolic token is used to
+determine its fixity, in the same way we expect a human reader to recognize
+them. For example, we want `a* - 4` to treat the `*` as a unary operator and the
+`-` as a binary operator, while `a * -4` treats `*` as a mathematical operation
+and `-` as the negative sign. Hence we can say that the whitespaces plays a
+really important role here, and we use some rules to avoid confusion:
+
+-   There can be no whitespace between a unary operator and its operand.
+-   The whitespace around a binary operator must be consistent: either there is
+    whitespace on both sides or on neither side.
+-   If there is whitespace on neither side of a binary operator, the token
+    before the operator must be an identifier, a literal, or any kind of closing
+    bracket (for example, `)`, `]`, or `}`), and the token after the operator
+    must be an identifier, a literal, or any kind of opening bracket (for
+    example, `(`, `[`, or `{`).
+
+## Details
+
+Symbolic tokens are intended to be used for widely-recognized operators, such as
+the mathematical operators `+`, `*`, `<`, and so on. Those used as operators
+would generally be expected to also be meaningful for some user-defined types,
+and should be candidates for being made overloadable once we support operator
+overloading.
+
+### Symbolic token list
+
+The following is the initial list of symbolic tokens recognized in a Carbon
+source file:
+
+| Token | Explanation                                                                                                |
+| ----- | ---------------------------------------------------------------------------------------------------------- |
+| `*`   | Indirection, multiplication, and forming pointers                                                          |
+| `&`   | Address-of or Bitwise AND                                                                                  |
+| `=`   | Assignment                                                                                                 |
+| `->`  | Return type and `p->x` equivalent to `(*p).x` (in C++)                                                     |
+| `=>`  | Match syntax                                                                                               |
+| `[]`  | Subscript                                                                                                  |
+| `()`  | Function call and function declaration                                                                     |
```

Comment:
```suggestion
| `()`  | Function call and function declaration and tuple literals                                                 |
```

Reply:
This was taken from [here](https://github.com/carbon-language/carbon-lang/pull/2519/files#r1086858109) and it was corrected by @josh11b so, not sure whether I should add it!

Reply:
#2519 started by trying to document "operator tokens", but we eventually decided that wasn't the way to go. For example, `->` is sometimes an operator (in `x->y`) and sometimes not (in `fn F() -> bool`). Instead, we decided that the most useful thing was to consider this from a lexing perspective, which does not distinguish whether the token is an operator, except whether it has whitespace on each side. As long as the presence or absence of whitespace information is preserved, lexing can treat the tokens uniformly. See the discussion here, which is very relevant to this PR:
https://discord.com/channels/655572317891461132/709488742942900284/1067599248813740072

With this shift to talking about symbolic tokens, it is fair to say `()` is also used for tuple literals.

Reply:
Really sorry for the late reply, got held up in urgent work! I will add these!